tests/introspection/ApiAvailabilityTest.cs

```diff
@@ -160,6 +160,9 @@ void CheckIntroduced (Type t, AvailabilityBaseAttribute ta, MemberInfo m)
 		}
 
 		[Test]
+#if NET
+		[Ignore ("Requires attributes update - see status in ...")]
```

Comment:
Missing link to GH issue?

Reply:
oops, it was not yet created at the time
I'll fix this along other tests that needs to be updated (there seems to be more than I thought)